backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -70,4 +77,20 @@ public void testGetWaitingTime() throws Exception {
         
         assertEquals("Waiting Time per customer is", WAITING_TIME_MINS, waitingTime);
     }
+
+    @Test
```

Comment:
Follow the AAA pattern
https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80